mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/screens/SetUpUPiPinScreen.kt

```diff
@@ -0,0 +1,106 @@
+package org.mifos.mobilewallet.mifospay.bank.screens
+
+import SetUpUpiScreenContent
+import android.app.Activity
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.tooling.preview.Preview
+import com.mifos.mobilewallet.model.domain.BankAccountDetails
+import org.mifos.mobilewallet.mifospay.common.Constants
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SetupUpiPinScreen(
+    bankAccountDetails: BankAccountDetails,
+    type: String,
+    index: Int,
+    correctlySetingUpi: (String) -> Unit
+) {
+    val context = LocalContext.current
+    Scaffold(
+        topBar = {
+            TopAppBar(
+                title = {
+                    Text(text = when (type) {
+                        Constants.SETUP -> Constants.SETUP_UPI_PIN
+                        Constants.CHANGE -> Constants.CHANGE_UPI_PIN
+                        Constants.FORGOT -> Constants.FORGOT_UPI_PIN
+                        else -> ""
+                        }
+                    )
+                },
+                navigationIcon = {
+                    IconButton(onClick = {
+                        (context as? Activity)?.onBackPressed()
+                    })
+                    {
+                        Icon(Icons.Filled.ArrowBack, contentDescription = "Back")
```

Comment:
same here ,use defined icons in MifosIcons and use string res

Reply:
Seems ,ArrowBack is not defined in MifosIcons.
Should we go with this only ?

Reply:
Yes you can

Reply:
done as suggested in all . please get it merged if everything is OK.